libs/barista-components/sunburst-chart/src/sunburst-chart.scss

```
@@ -17,15 +19,16 @@ text {
   fill: var(--dt-sunburst-chart-slice-color);
 
   &:hover {
-    fill: var(
-      --dt-sunburst-chart-slice-color--hover,
-      var(--dt-sunburst-chart-slice-color)
-    );
+    opacity: $hover-opacity;
```

Single use variable can be omitted.
```suggestion
    opacity: 0.8;
```